_data/projects/Learning_Object_Oriented_Python.yml

```
@@ -0,0 +1,10 @@
+name: Learning-Object_Oriented-Python
+desc: This repository walks you through the Object Oriented Programming in Python.
+site: https://www.github.com/josharsh/Learning-Object-Oriented-Python
+tags:
+- python
+- documentation
+- Object-Oriented-Python
+upforgrabs:
+  name: good first issue
+  link: https://github.com/josharsh/Learning-Object-Oriented-Python/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

A more succinct label that helps our automation to generate stats for this project:

```suggestion
  link: https://github.com/josharsh/Learning-Object-Oriented-Python/labels/good%20first%20issue
```